### PR TITLE
[Kofta] Add Australian translation

### PR DESCRIPTION
**Pull request by nihaals**

### CODE REVIEW

File: kofta/src/app/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
 
     /* Other languages */
     { value: "en-PIRATE", label: "🏴‍☠️ Pirate" },
+    { value: "au", label: "🇦🇺 uɐᴉꞁɐɹʇsnⱯ" }, // Australian
```
Comment (nihaals) [inline kofta/src/app/components/LanguageSelector.tsx]: Not sure if it makes more sense to have the flag at the end instead of the beginning

**Comment by amitojsingh366** [comment]
😂😂

**Comment by amitojsingh366** [comment]
I see what you're tying to do here but all languages need to follow the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes. And unfortunately `au` is an unassigned code there. Not sure if unassigned means we can use it here...

**Comment by amitojsingh366** [comment]
> 
> 
> I see what you're tying to do here but all languages need to follow the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes. And unfortunately `au` is an unassigned code there. Not sure if unassigned means we can use it here...

But i think if `en-PIRATE` is a thing then ig this can also be valid

**Comment by nihaals** [comment]
> I see what you're tying to do here but all languages need to follow the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes. And unfortunately `au` is an unassigned code there. Not sure if unassigned means we can use it here...

I think that's what the `Other languages` section is for, pretty much all of the languages there are similar to this one.

I did make sure I wasn't actually intruding on an ISO 639-1 code, but as you said it's not listed, so it should be safe to class this as an "other" language.

**Comment by ermalsh** [comment]
Apps using i18n commonly use this for en-*: **en-AU**, en-CA, en-GB, en-IN, en-US.

**Comment by amitojsingh366** [comment]
@nihaals are you still working on this?

**Comment by nihaals** [comment]
@amitojsingh366 Yes sorry, haven't been able to update it yet

**Review by amitojsingh366 — APPROVED** [review]
Looks good!

**Pull request merged**